src/test/config/setupopts.test.json

```diff
@@ -14,5 +14,13 @@
       "defaultNetwork":{
          "interface_name":"eth0"
       }
+   },
+   "network_info":{
```

Comment:
I think it `network_info` takes a map of `stirng`: `<Network>`. I think this is a direct <Network>